source/_integrations/keyboard_remote.markdown

```diff
@@ -22,10 +22,25 @@ keyboard_remote:
 
 {% configuration %}
 type:
-  description: Possible values are `key_up`, `key_down`, and `key_hold`. Be careful, `key_hold` will fire a lot of events.
+  description: Possible values are `key_up`, `key_down`, and `key_hold`. Be careful, `key_hold` will fire a lot of events.  This can be a list of types.
   required: true
   type: string
-device_description:
+emulate_key_hold:
+  description: Emulate key hold events when key is held down.  (Some input devices do not send these otherwise.)
+  required: false
+  type: boolean
+  default: false
+emulate_key_hold_delay:
+  description:  Number of milliseconds to wait before sending first emulated key hold event
+  required: false
+  type: positive_int
```

Comment:
`positive_int` is not a valid type for the documentation, use `integer` instead.

Reply:
This has been changed to float in the end.